docs/pages/setup/guides/joining-nodes-aws-ec2.mdx

```diff
@@ -207,6 +212,15 @@ proxy_service:
 
 ## Step 4/4. Launch your Teleport node
 
+<Admonition
+  type="note"
+  title="IMPORTANT"
+>
+  The data directory (`/var/lib/teleport` by default) should be empty prior to launching the
+  node.  An error will occur if this node had previously joined by another method (e.g. token or IAM) and is now switching to the EC2 method. 
```

Comment:
What is the nature of the error? Will running `teleport start` fail with a message, or will this be something more mysterious?

Reply:
@ptgott modified phrasing